GitHub-Org-Management-Policy.md

```diff
@@ -0,0 +1,70 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation GitHub Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the direction
+of the Node.js Technical Steering Committee (TSC) and Node.js Foundation
+Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The [Node.js admin repository](https://github.com/nodejs/admin) will serve as
+the central location for managing Node.js GitHub Organization administrative
+activities. Only Node.js GitHub Organization owners, TSC members, and Community
+Committee members will have write permissions to the Node.js admin repository.
+
+## Organization Roles
+
+### Owners
+
+TSC members and the Chair of the Community Committee are the only individuals granted
```

Comment:
there can be multiple chairs of the commcomm

Reply:
Agreed, I thought that was a detail we could leave out with the the assumption it would include chairs in the exception case (which of course is what we have now).

Reply:
that works!